scripts/trainer.py

```diff
@@ -57,17 +57,18 @@ def go(self, args):
 
         logger.info("Training done")
 
-        model_file_name = f"{model_name}model"
-        assert os.path.exists(model_file_name)
-        zstd_compress(model_file_name)
+        model_directory = f"{model_name}model"
+        assert os.path.exists(model_directory)
+
+        create_tar_zst(f"{model_directory}.tar.zst")
 
         logger.info("Model compressed")
 
         if model_obj.store_dataset:
-            assert os.path.exists(f"{model_file_name}_data_X")
-            zstd_compress(f"{model_file_name}_data_X")
-            assert os.path.exists(f"{model_file_name}_data_y")
-            zstd_compress(f"{model_file_name}_data_y")
+            assert os.path.exists(f"{model_directory}_data_X")
+            zstd_compress(f"{model_directory}_data_X")
+            assert os.path.exists(f"{model_directory}_data_y")
+            zstd_compress(f"{model_directory}_data_y")
```

Comment:
It is a bit confusing to use `model_directory` because they have nothing to do with the model directory:
```suggestion
            assert os.path.exists(f"{model_name}model_data_X")
            zstd_compress(f"{model_name}model_data_X")
            assert os.path.exists(f"{model_name}model_data_y")
            zstd_compress(f"{model_name}model_data_y")
```

Reply:
> It is a bit confusing to use `model_directory` because they have nothing to do with the model directory:

Ah right! I thought so as well 🤓